source/reference/methods.txt

```diff
@@ -251,10 +251,111 @@ Collection Methods
 
      - Modifies multiple documents in a collection.
 
+Bulk Operation Methods
+----------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+
+   * - :method:`db.collection.initializeOrderedBulkOp()`
+
+     - Initializes and returns a new :method:`Bulk()` operations builder
+       for a collection. The builder constructs an ordered list of write
+       operations that MongoDB executes in bulk.
+
+   * - :method:`db.collection.initializeUnorderedBulkOp()`
+
+     - Initializes and returns a new :method:`Bulk()` operations builder
+       for a collection. The builder constructs an unordered list of write
+       operations that MongoDB executes in bulk.
+   
+   * - :method:`Bulk()`
+
+     - Creates a bulk operations builder used to construct a list of write
+       operations to perform in bulk for a single collection. To instantiate
+       the builder, use either the :method:`db.collection.initializeOrderedBulkOp()`
+       or the :method:`db.collection.initializeUnorderedBulkOp()` method.
+
+   * - :method:`Bulk.execute()`
+
+     - Executes the list of operations built by the :method:`Bulk()`
+       operations builder.
+
+   * - :method:`Bulk.find()`
+
+     - Specifies a query condition for an update or a remove operation.
+
+   * - :method:`Bulk.find.arrayFilters()`
```

Comment:
The ticket says that `arrayFilters` and `collation` are blocked. Do they work in mongosh? If not, I think we should hold off on documenting them here.

Reply:
Whoops, my mistake, removed those two methods.